base/ltexpl.dtx

```diff
@@ -188,18 +188,26 @@
         \@gobble
 %</latexrelease>
       }%
-      {%
-        \input expl3.ltx
-        \ifdefined\NewDocumentCommand
-        \else
-          \IfFileExists{xparse.ltx}
-            {\input xparse.ltx }
-            {}%
-         \fi
-      }%
+      {\input expl3.ltx }%
   }
 %<latexrelease>\EndIncludeInRelease
 %<latexrelease>
+%    \end{macrocode}
+% To support roll-forward for the case where \textsf{xparse} is fully
+% integrated into the kernel, we do not need to repeate the complex test
+% above as we can simply look for the marker command.
```

Comment:
and that work also for rolling back to that date?

Reply:
Er, I *think* so